src/sql/error_handler.py

```diff
@@ -47,7 +47,7 @@ def _detailed_message_with_error_type(error, query):
         "invalid sql",
     ]
     not_found_substrings = [
-        "does not exist",
+        "table with name",
```

Comment:
This phrase can be used in other types of errors too so it might be misleading. Maybe we should check for the entire phrase `table with name <name> dies not exist` using regex?

Reply:
[Fixed using regex](https://github.com/ploomber/jupysql/pull/912/files#diff-909f978f8e88042aeb731463242ae800a3d7e760ba0eb817c51edf830cd3e7e7R49-R53) - also take a look [here](https://github.com/ploomber/jupysql/pull/912/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97L404-R409)